editor.html

```diff
@@ -277,18 +277,18 @@ <h2><i class="fa fa-unlock-alt"></i> <strong>{{ title }}</strong></h2>
                     </div>
                 </div>
                 <div id="small-icons" class="vbox">
-                    <a class="roundbutton holder zoomer action" tabindex="9" id="command-zoom-in" href="#" title="Zoom in">
+                    <a aria-label="zoom in" class="roundbutton holder zoomer action" tabindex="9" id="command-zoom-in" href="#" title="Zoom in">
                         <div class="status-icon"><i class="fa fa-search-plus"></i></div>
                     </a>
-                    <a class="roundbutton holder zoomer action" tabindex="10" id="command-zoom-out" href="#" title="Zoom out">
+                    <a aria-label="zoom out" class="roundbutton holder zoomer action" tabindex="10" id="command-zoom-out" href="#" title="Zoom out">
                         <div class="status-icon"><i class="fa fa-search-minus"></i></div>
                     </a>
                 </div>
                 <div id="small-icons" class="vbox">
-                    <a class="roundbutton holder zoomer action" tabindex="11" id="command-snippet" href="#" title="Click to select a snippet (code shortcut)">
+                    <a aria-label="show snippets (code shortcuts)" class="roundbutton holder zoomer action" tabindex="11" id="command-snippet" href="#" title="Click to select a snippet (code shortcut)">
                         <div class="status-icon"><i class="fa fa-cogs"></i></div>
                     </a>
-                    <a class="roundbutton holder zoomer action" tabindex="12" id="command-language" href="#" title="Change the language">
+                    <a aria-label="change the language" class="roundbutton holder zoomer action" tabindex="12" id="command-language" href="#" title="Change the language">
```

Comment:
We will also need to update all these aria-label strings when switching the language, which is done in `translations.translateEmbedStrings()`.

Reply:
do we also need to update en.js with the strings?

Reply:
They would be the same as the `title`, no?

Reply:
Yep, so would it be $(value).attr('aria-label', buttons[button_id]['title']) instead of $(value).attr('aria-label', buttons[button_id]['aria-label'])?

Reply:
Yes, let's also edit the default text here to be the same as the title.